hack/release/wizard/releaseWizard.yaml

```diff
@@ -690,45 +697,31 @@ groups:
         comment: "We are only committing with the new version, we will not push to the release branch until after the vote has succeeded."
         logfile: commit.log
         stdout: true
-  - !Todo
-    id: build_rc
-    title: Build the release candidate
-    depends:
-      - rc_versions
-      - gpg
-    vars:
-      rc_folder: "solr-operator-{{ release_version }}-RC{{ rc_number }}-rev{{ build_rc.git_rev }}"
-      local_keys: '{% if keys_downloaded %} --local-keys "{{ [config_path, ''KEYS''] | path_join }}"{% endif %}'
-    commands: !Commands
-      root_folder: '{{ git_checkout_folder }}'
-      commands_text: |-
-        In this step we will build the RC artifacts.
-      confirm_each_command: false
-      env:
-        TAG: "{{ release_version }}-rc{{ rc_number }}"
-        ARTIFACTS_DIR: "{{ [dist_file_path, rc_folder] | path_join }}"
-        APACHE_ID: "{{ gpg.apache_id }}"
-        GPG_KEY: "{{ gpg_key }}"
-      commands:
-        - !Command
-          cmd: make build-release-artifacts
-          comment: "NOTE: Remember to type your GPG pass-phrase at the prompt!"
-          logfile: build_rc.log
-          tee: true
+      - !Command
+        vars:
+          rc_folder: 'solr-operator-{{ release_version }}-RC{{ rc_number }}-rev{{ current_git_rev }}'
+        env:
+          TAG: "{{ release_version }}-rc{{ rc_number }}"
+          ARTIFACTS_DIR: "{{ [dist_file_path, rc_folder] | path_join }}"
+          APACHE_ID: "{{ gpg.apache_id }}"
+          GPG_KEY: "{{ gpg_key | default('', True) }}"
+        cmd: make build-release-artifacts
+        comment: "NOTE: Remember to type your GPG pass-phrase at the prompt!"
+        logfile: build_rc.log
+        tee: true
   - !Todo
     id: smoke_tester
     title: Run the smoke tester
     depends: build_rc
     vars:
-      dist_folder: solr-operator-{{ release_version }}-RC{{ rc_number }}-rev{{ build_rc.git_rev | default("<git_rev>", True) }}
-      dist_path: '{{ [dist_file_path, dist_folder] | path_join }}'
+      dist_path: '{{ [dist_file_path, build_rc.rc_folder] | path_join }}'
       docker_image: 'apache/solr-operator:{{ release_version }}-rc{{ rc_number }}'
     commands: !Commands
       root_folder: '{{ git_checkout_folder }}'
       commands_text: Here we'll smoke test the release by 'downloading' the artifacts, running the tests, validating GPG signatures etc.
       commands:
       - !Command
-        cmd: ./hack/release/smoke_test/smoke_test.sh -v "{{ release_version }}" -s "{{ build_rc.git_rev | default("<git_rev>", True) | truncate(7) }}" -i "{{ docker_image }}" -l "{{ dist_path }}"
+        cmd: ./hack/release/smoke_test/smoke_test.sh -v "{{ release_version }}" -s "{{ build_rc.git_sha }}" -i "{{ docker_image }}" -l "{{ dist_path }}"
```

Comment:
The | `default()` clause is there for the export-to-HTML to work, i.e. it will output `<git_rev>` if git_sha is not yet recorded, thus you can generate a generic HTML checklist not tied to a specific release.

Reply:
ahhhh ok, I was very curious about that part. Makes sense. I will use the `default()` for all the stored variables I use throughout.